src/main/java/com/google/api/codegen/php/PhpDiscoveryContext.java

```diff
@@ -52,13 +54,31 @@
           .put(Field.Kind.TYPE_BOOL, "false")
           .put(Field.Kind.TYPE_INT32, "0")
           .put(Field.Kind.TYPE_UINT32, "0")
-          .put(Field.Kind.TYPE_INT64, "0")
-          .put(Field.Kind.TYPE_UINT64, "0")
           .put(Field.Kind.TYPE_FLOAT, "0.0")
           .put(Field.Kind.TYPE_DOUBLE, "0.0")
+          // As a work-around in Javascript, Disovery importor treats number format strings
```

Comment:
s/importor/importer/

Could you clarify what you mean by "As a workaround in Javscript"?